### PR TITLE
database backup tasks

### PR DESCRIPTION
**Pull request by samsonasu**

### CODE REVIEW

File: lib/tasks/db.rake [hunk 1]
```diff
@@ -0,0 +1,37 @@
+namespace :db do
+  desc 'Capture a database backup to tmp/latest.dump'
+  task :backup do
+    sh "pg_dump \
+        --verbose -Fc \
+        #{hostname} #{username} -d #{db_config("database")} \
+        -f #{Rails.root.join('tmp', 'latest.dump')}"
```
Comment (samsonasu) [inline lib/tasks/db.rake]: change this to a timestamped filename and symlink it?

File: lib/jeffries_tube/capistrano/capistrano3.rb [hunk 2]
```diff
@@ -29,3 +29,15 @@ def run_interactively(command)
...
+
+namespace :db do
+  desc "Capture a database snapshot"
+  task :backup do
+    on roles(:db), primary: true do |host|
+      within release_path do
+        execute :rake, "db:backup", "RAILS_ENV=#{fetch(:stage)}"
+      end
+    end
+
```
Comment (samsonasu) [inline lib/jeffries_tube/capistrano/capistrano3.rb]: add backup:fetch to fetch the latest from the server via scp

**Comment by samsonasu** [comment]
would be super awesome to add rolling backup functionality - save last 5 or 10 or whatever to somewhere.  Also to upload to AWS maybe? We'd need to find a place to put amazon credentials

File: lib/tasks/db.rake [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  end
+
+  def username
+    username = db_config('username')
+    if (username)
+      "-U #{username}"
+    else
+      ""
+    end
+  end
```
Comment (boatrite) [inline lib/tasks/db.rake]: A newline really wants to be here.

**Comment by boatrite** [comment]
:anchor:

**Pull request merged**